portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -49,6 +53,16 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
             commentEntity.setProperty("message", message);
             commentEntity.setProperty("timestamp", timestamp);
 
+
+            Document doc =
+              Document.newBuilder().setContent(message).setType(Document.Type.PLAIN_TEXT).build();
+				LanguageServiceClient languageService = LanguageServiceClient.create();
+				Sentiment sentiment = languageService.analyzeSentiment(doc).getDocumentSentiment();
+				double sentimentScore = (double) sentiment.getScore();
+				languageService.close();
```

Comment:
This block of code feels like it could use a comment as it's different from the rest of the surrounding code and that's an indication to me that you might want to extract this to a smaller, more focused, clearly-named private helper method. This can help make the code more readable.

Reply:
I originally had this, but I changed it back for some reason. Good idea :-)